deploy/downstream-prep.sh

```diff
@@ -7,7 +7,7 @@ rm -rf deploy/olm-catalog/mig-operator/stable deploy/olm-catalog/mig-operator/la
 
 # 1.0.0 Downstream CSV Changes
 sed -i s,quay.io,image-registry.openshift-image-registry.svc:5000,g                              deploy/olm-catalog/mig-operator/v1.0.0/mig-operator.v1.0.0.clusterserviceversion.yaml
-sed -i s,ocpmigrate,rhcam,g                                                                      deploy/olm-catalog/mig-operator/v1.0.0/mig-operator.v1.0.0.clusterserviceversion.yaml
+sed -i s,ocpmigrate,rhcam-1.0,g                                                                  deploy/olm-catalog/mig-operator/v1.0.0/mig-operator.v1.0.0.clusterserviceversion.yaml
```

Comment:
is 'rhcam-1.0' what is needed or 'rhcam-1-0'
COMET disallows usage of '.', not sure if this was to reflect the distribution request in COMET for 'rhcam-1-0'

Reply:
Thank you. You are correct.